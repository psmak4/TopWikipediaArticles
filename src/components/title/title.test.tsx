import { render, screen } from '@testing-library/react'
import Title from './title'

describe('Title component', () => {
	test('renders the component correctly', () => {
		render(<Title />)
		const titleElement = screen.getByText('Top Wikipedia articles')
		expect(titleElement).toBeInTheDocument()
		expect(titleElement).toHaveClass('title')
	})

	test('matches snapshot', () => {
		const { asFragment } = render(<Title />)
		expect(asFragment()).toMatchSnapshot()
	})
})
