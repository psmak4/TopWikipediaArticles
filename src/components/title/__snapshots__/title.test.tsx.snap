// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component matches snapshot 1`] = `
<DocumentFragment>
  <h1
    class="title"
  >
    Top Wikipedia articles
  </h1>
</DocumentFragment>
`;
